Picker/Picker.js

```diff
@@ -160,6 +160,21 @@ const PickerBase = kind({
 		 */
 		noAnimation: PropTypes.bool,
 
+		/**
+		 * Sets rules to show the indicator and to determine the user interaction of the control
+		 * at a horizontal joined Picker case.
+		 *
+		 * When `false`, the indicator are shown and allow the user to use the Enter keys to adjust the picker's value.
```

Comment:
```suggestion
		 * When `false`, the indicators are shown and allow the user to use the Enter keys to change the picker's value.
```
But english is not my area of expertise.